packages/core/src/sass/app/modules/cashier.scss

```
@@ -354,23 +354,39 @@
         }
     }
     &__account-switch-icon {
-        background-image: url('~Images/app/cashier/account_switcher.png');
-        width: 400px;
-        height: 360px;
+        box-sizing: border-box;
+        width: 299px;
+        height: 200px;
         background-repeat: no-repeat;
         background-size: 100%;
-        margin: 2.4rem auto;
+        margin: 0 auto 1rem;
+
+        &--light {
+            background-image: url('~Images/app/cashier/cashier-demo-light.svg');
+        }
+        &--dark {
+            background-image: url('~Images/app/cashier/cashier-demo-dark.svg');
+        }
 
         @include mobile {
             width: 100%;
             margin: 2.4rem 0 0;
             height: 320px;
         }
     }
+    &__account-switch-text {
+        color: var(--text-loss-danger);
+        cursor: pointer;
+        font-weight: bold;
+    }
     &__virtual {
         &-header {
             margin-top: 0;
-            margin-bottom: 0.8rem;
+            margin-bottom: 1.6rem;
+            color: var(--text-prominent);
+            font-weight: bold;
+            font-size: 1.6rem;
+            line-height: 1.5;
```

```suggestion
@include typeface(--title-left-bold-prominent);
```

Can we leverage the typeface mixin here?